src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/StateResetInProcess.cs

```diff
@@ -98,15 +98,28 @@ public async Task ResetHostSettingsAsync(CancellationToken cancellationToken)
             await TestServices.Editor.SetUseSuggestionModeAsync(forDebuggerTextView: true, true, cancellationToken);
             await TestServices.Editor.SetUseSuggestionModeAsync(forDebuggerTextView: false, false, cancellationToken);
 
+            // Make sure responsive completion doesn't interfere if integration tests run slowly.
+            await DisableResponsiveCompletion(cancellationToken);
+
+            // Disable frozen partial semantics
+            var workspace = await TestServices.Shell.GetComponentModelServiceAsync<VisualStudioWorkspace>(cancellationToken);
+            var hook = workspace.Services.GetRequiredService<IWorkspacePartialSolutionsTestHook>();
+            hook.IsPartialSolutionDisabled = true;
```

Comment:
This part shouldn't need to change

Reply:
Why? We'd like to disable frozen partial since we don't wait for compilation to be available.

Reply:
Mainly because I wasn't able to show any improvement from it when testing to see if it was needed, and leaving it enabled is a better match to what we actually ship.